### ISSUE TITLE
Add rustup

### ISSUE THREAD

**Issue by ArchGuyWu** [msys2/MINGW-packages#16145]
### Package name

rustup

### Brief description of package

rustup is an installer for
 the systems programming language Rust

### URL for package's homepage

https://rustup.rs/

### MINGW environments where you need the package

- [X] MINGW64
- [ ] MINGW32
- [X] UCRT64
- [X] CLANG64
- [ ] CLANG32
- [X] CLANGARM64

**Comment by ArchGuyWu** [comment]
It would be useful for those who need multiple targets.

**Comment by Kreijstal** [comment]
Yeah this would be nice, I guess you'd have to compile rustup yourself

**Comment by 500-internal-server-error** [comment]
For now I have a [`PKGBUILD` here](https://github.com/500-internal-server-error/aur-msys2/blob/main/mingw-w64-rustup/PKGBUILD), mostly taken from Arch with some modifications to better fit in MSYS2 (notably replacing symlinks to multicall binaries with hardlinks, to maintain the small package size without requiring modifications to a stock Win8+ install).

**Comment by 500-internal-server-error** [comment]
https://github.com/msys2/MINGW-packages/pull/20989 has been merged, this issue can be closed.
